src/Controller/TalksController.php

```diff
@@ -34,7 +55,7 @@ public function getTalkStarred(Request $request, PDO $db)
     {
         $this->setDbAndRequest($db, $request);
         $talk_id = $this->getItemId($request);
-        $mapper = $this->getMapper('talk');
+        $mapper  = $this->getMapper('talk');
```

Comment:
What is the benefit to aligning assignment operators? If we were to add a `$longerName` var on the next line, then that would require updating lines 57 & 58 also, and that would make tracing through the git history more difficult.